src/selectors/per-thread/stack-sample.js

```diff
@@ -217,7 +217,7 @@ export function getStackAndSampleSelectorsPerThread(
   );
 
   const getStackTimingByDepth: Selector<StackTiming.StackTimingByDepth> = createSelector(
-    threadSelectors.getFilteredThread,
+    threadSelectors.getFilteredSamplesForCallTree,
```

Comment:
Question: Does the stack table even provide anything useful for memory information? It's definitely good to make sure what we have is at least correct. I'd have to poke around a bit with memory profiles to decide.

Reply:
You mean the stack chart?

Yeah that's a good question, I'm not so sure either.

Reply:
Yes, stack chart, sorry. I believe when I originally implemented this the stack chart was worthless for memory, and this patch essentially re-activates the stack chart for this use case. I'm guessing I didn't write an explicit test for this behavior. I would validate that this change is actually one that's good to do, and that the stack chart makes sense for all of the allocation types.

Reply:
I've been thinking, and I believe the stack chart is (currently) the only place where we see the memory (de)allocations happening chronologically (besides the memory track which comes from the counter data). We should likely change this in the future and have this appear in the timeline, but for now I believe this has some value. And with the select box being present we're explicit about what's being displayed, so this shouldn't be confusing, I believe.

Reply:
Makes sense, I agree!

Reply:
The tooltip is still time-based though, I missed that. I think I'll fix that in a follow-up.

Reply:
And actually I'm not so sure the data displayed in the stack chart is currently accurate, I'll need to debug this more. 😕